.azure-pipelines/all.yml

```diff
@@ -198,6 +198,9 @@ jobs:
     - checkName: linux_proc_extras
       displayName: Linux proc extras
       os: linux
+    - checkName: mapr
```

Comment:
I think you may have done something weird when merging/rebasing master.
You can fix individual files with:

```
git pull
git checkout origin/master -- <path/to/file>
```
And then commit amend and force push